.gitignore

```diff
@@ -50,8 +50,6 @@ java/gradle
 java/.gradle
 java/hs_*.log
 onnxruntime/python/version_info.py
-/orttraining/orttraining/eager/ort_aten.g.cpp
```

Comment:
this will be removed. I added it to help show the effect of the changes. Only the generator changes will be kept after folks take a look.